Ansible/collection/jfrog/ansible/galaxy.yml

```diff
@@ -45,13 +48,13 @@ tags: []
 dependencies: {}
 
 # The URL of the originating SCM repository
-repository: http://example.com/repository
+repository: "https://github.com/jfrog/JFrog-Cloud-Installers/"
 
 # The URL to any online docs
-documentation: http://docs.example.com
+documentation: "https://www.jfrog.com/confluence/display/JFROG/"
```

Comment:
I think this should be URL to ansible doc

Reply:
Changed to Ansible collection README.